### PR TITLE
chore(evm): Add meta skeleton with manual seal

### PR DESCRIPTION
**Pull request by surangap**
#### What this PR does / why we need it:
Adds initial code required for meta node with manual seal.
#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

**Review by DieHard073055 — APPROVED** [review]
LGTM!
was able to build and run it. Thanks!

File: runtimes/meta-runtime/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+parity-scale-codec = { version = "2.0", default-features = false, features = ["derive"] }
+scale-info = { version = "1.0", default-features = false, features = ["derive"] }
+serde = { version = "1.0", optional = true, features = ["derive"] }
+
+# Substrate packages
+frame-executive = { default-features = false, git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
```
Comment (canonbrother) [inline runtimes/meta-runtime/Cargo.toml:12]: suggestion: just need to specify either `tag` or `version`

File: Cargo.toml [hunk 2]
```diff
@@ -1,4 +1,10 @@
 [workspace]
 members = [
-    "example/example",
-]
\ No newline at end of file
+	"pallets/meta-consensus",
+	"runtimes/meta-runtime",
+	"nodes/meta-node",
+	"example/example",
```
Comment (canonbrother) [inline Cargo.toml:6]: suggestion:
```suggestion
	"pallets/*",
	"runtimes/*",
	"nodes/*",
	"example/*",
```

**Review by fuxingloh — APPROVED** [review]
Build, ran, tested locally, and looked through all the files, it look's good. Let's get this in and iterate from there 🚀

**Pull request merged**